src/client/modules/ExportWins/Form/WinDetailsStep.jsx

```diff
@@ -1,21 +1,225 @@
+      <H3 data-test="step-heading">Win details</H3>
+      <StyledHintParagraph data-test="hint">
+        The customer will be asked to confirm this infomation.
+      </StyledHintParagraph>
+      <ResourceOptionsField
+        name="country"
+        id="country"
+        label="Destination country"
+        required="Choose a destination country"
+        resource={CountriesResource}
+        field={FieldTypeahead}
+      />
+      <FieldDate
+        name="date"
+        format="short"
+        label="Date won"
+        hint={`For example ${month} ${year}, date of win must be in the last 12 months.`}
+        required="Enter the win date"
+        invalid="Enter a valid date"
+      />
+      <FieldTextarea
+        name="description"
+        label="Summary of the support given"
+        hint="Outline what had the most impact or would be memorable to the customer in less than 100 words."
+        required="Enter a summary"
+        invalid={`Summary must be ${MAX_WORDS} words or less`}
+        maxWords={MAX_WORDS}
+      />
+
+      <FieldInput
+        type="text"
+        name="name_of_customer"
+        label="Overseas customer"
+        required="Enter the name of the overseas customer"
+        placeholder="Add name"
+      />
+
+      <FieldCheckboxes
+        name="name_of_customer_confidential"
+        hint="Check this box if your customer has asked for this not to be public (optional)."
+        options={[
+          {
+            value: OPTION_YES,
+            label: 'Confidential',
+          },
+        ]}
+      />
+
+      <FieldInput
+        type="text"
+        name="business_type"
+        label="Type of business deal"
+        required="Enter the type of business deal"
+        hint="For example: export sales, contract, order, distributor, tender / competition win, joint venture, outward investment."
+        placeholder="Enter a type of business deal"
+      />
+
+      <FieldCheckboxes
+        name="win_type"
+        legend="Type of win"
+        required="Choose at least one type of win"
+        options={winTypeOptions.map((option) => ({
+          ...option,
+          ...(option.value === winTypes.EXPORT && {
+            children: (
+              <WinTypeValues
+                label="Export value over the next 5 years"
+                name="export_win"
+                values={values}
+              />
+            ),
+          }),
+          ...(option.value === winTypes.BUSINESS_SUCCESS && {
+            link: (
+              <StyledDetails
+                summary="What is business success?"
+                data-test="business-success-details"
+              >
+                <p>Business success is defined as:</p>
+                <UnorderedList listStyleType="bullet">
+                  <ListItem>
+                    the exchange of ownership of goods/services from a
+                    subsidiary of an eligible UK company to a non_UK resident.
```

Comment:
```suggestion
                    subsidiary of an eligible UK company to a non-UK resident.
```
I'm not sure there should be an underscore here